R/Vietnam.R

```diff
@@ -0,0 +1,124 @@
+#' Vietnam Class for downloading, cleaning and processing
+#' notification data
+#'
+#' @description Information for downloading, cleaning
+#'  and processing covid-19 region data for Vietnam.
+#'
+# nolint start
+#' @source \url{https://github.com/biocyberman/covidregionaldata/}
+# nolint end
```

Comment:
This should likely point to the same thing as the `source_url` field (or a human-readable entry point into the data). `nolint` is only necessary when the lines are longer than what `lintr` likes - probably around 75.

Reply:
Fixed that. Changed to the same as `source_url` and removed lint config.